README.md

```diff
@@ -99,6 +99,19 @@ $ curl https://specs.frictionlessdata.io/schemas/data-resource.json | jsv --outp
 …
 ```
 
+#### Exporting to Python
+
+```console
+$ curl https://specs.frictionlessdata.io/schemas/data-resource.json | jsv --output py
+dataset_metadata = {
+    "profile": "data-resource",  # The profile of this descriptor.
+    # [example] "profile": "tabular-data-package"
+    # [example] "profile": "http://example.com/my-profiles-json-schema.json"
+    "name": "my-nice-name",  # An identifier string. Lower case characters with `.`, `_`, `-` and `/` are allowed.
+    "path": ["file.csv","file2.csv"],  # A reference to the data for this resource, as either a path as a string, or an array of paths as strings. of valid URIs.
```

Comment:
The default `path` value should be a single string i think. BTW these values can come from default or an example ... (e.g. pick first example and use that value).

Reply:
>  these values can come from default or an example ... (e.g. pick first example and use that value).

That is exactly how it works. That [`["file.csv","file2.csv"]` is the first example of the `path` instance](https://github.com/datopian/jsv/blob/373f6635061fabe636a87227a71b758e135ce2b6/test/fixtures/data-resource.json#L74). It might be confusing because the nested fields under `oneOf`have their own examples, but, anyway… this is the first example for the `path`. Should we change the schema?